src/apps/backend/modules/comment/internal/comment-util.ts

```diff
@@ -0,0 +1,19 @@
+import { Comment } from '../types';
+import { CommentDB } from './store/comment-db';
+
+export function convertCommentDBToComment(commentDb: CommentDB): Comment {
+
+  const comment = new Comment();
+  
+  comment.id = commentDb._id.toString();
+  comment.account = commentDb.account.toString();
```

Comment:
Could you handle both account cases? As account can be objected as well as Object< Account >